src/main/java/com/github/manolo8/darkbot/modules/utils/PortalJumper.java

```diff
@@ -44,15 +47,22 @@ public void jump(Portal target) {
             if (!gm.group.isValid() || gm.group.size < minGroupSize) return;
         }
 
-        hero.jumpPortal(target);
```

Comment:
Try to keep this up here, if you move it down, jumping occurs after clicking which can lead to more desyncs

Reply:
there isn't a synchronous block to cause that, 1ms doesn't cause a desync

Reply:
you can't make assumptions about synchronism on the call, they're undefined behavior and api-dependant, however, it'd be better if they were sync in the future which is why code should make a best effort for that behavior

Reply:
reverted to original position